source/production/F0.Minesweeper.Logic.Abstractions/LocationShuffler.cs

```diff
@@ -3,5 +3,7 @@ namespace F0.Minesweeper.Logic.Abstractions
 	public enum LocationShuffler
 	{
 		GuidLocationShuffler,
+		RandomOrderLocationShuffler,
+		FisherYatesShuffler
```

Comment:
Inconsistent naming:
2 end in `LocationShuffler`, 1 ends in `Shuffler`.
Should we elide the `LocationShuffler`/`Shuffler` suffix, since the `enum` already carries that name?

Reply:
We decided to remove the suffix all together, since the `enum` name already carries it.